# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/test/marionette/test_private_repository_cleanup.py
# Commit: 6180f2e753bd
# Full Hash: 6180f2e753bd2efb4905c6a70eb6607144807d5b
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-11 10:49:37
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Add support for initializing corresponding origin to QuotaManager::OpenClientDirectory; r=devtools-reviewers,nchevobbe
#   
#   QuotaManager::OpenClientDirectory can now get initialization status of
#   corresponding origin and initialize the origin if it's needed.
#   
# ==============================================================================

diff -r f25dfc157ef7 -r 6180f2e753bd dom/quota/test/marionette/test_private_repository_cleanup.py
--- a/dom/quota/test/marionette/test_private_repository_cleanup.py	Fri Oct 11 00:06:08 2024 +0000
+++ b/dom/quota/test/marionette/test_private_repository_cleanup.py	Fri Oct 11 00:06:09 2024 +0000
@@ -40,7 +40,7 @@
         super(PrivateRepositoryCleanup, self).tearDown()
 
     def doStorageWork(self):
-        origin = self.marionette.absolute_url("")[:-1] + "^privateBrowsingId=1"
+        origin = "https://example.com^privateBrowsingId=1"
         assert self.initTemporaryOrigin("private", origin)
 
         self.ensureInvariantHolds(lambda _: os.path.exists(self.getPrivateRepository()))